### PR TITLE
Add .circleci/config.yml remove Travis CI

### PR DESCRIPTION
**Pull request by jeffmay**
My Travis CI account was not allowing me to run CI on this project, so I'm switching to Circle CI

### CODE REVIEW

**Review by paul-a-kennedy-rally — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    docker:
+      # specify the version you desire here
+      - image: circleci/openjdk:8
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
```
Comment (paul-a-kennedy-rally) [inline .circleci/config.yml:20]: Does this use shell expansion for this?  I figure you've tested it, but this isn't always a given...

Comment (jeffmay) [reply .circleci/config.yml:20]: This was the default template that CircleCI uses.

Comment (paul-a-kennedy-rally) [reply .circleci/config.yml:20]: Good example to show shell expansion is supported :)

**Review by darmbrus — APPROVED** [review]

**Pull request merged**